File: frontend/src/app/modules/admin/dto/cardto.ts
export class CarDTO{

    id?: number;
    name?: string;
    brand?: string;
    type?: string;
    color?: string;
    year?: string;
    transmission?: string;
    price?: number;
    image?: File;
    returnedImage?: string;

    constructor(name: string, brand: string, type: string, color: string, year: string, transmission: string, price: number, image: File){

        this.name = name;
        this.brand = brand;
        this.type = type;
        this.color = color;
        this.year = year;
        this.transmission = transmission;
        this.price = price;
        this.image = image;
    }
}

File: backend/src/main/java/com/programmingtest/Car_Rental/dto/CarDTO.java
package com.programmingtest.Car_Rental.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.*;
import org.springframework.web.multipart.MultipartFile;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CarDTO {

    private long id;

    private String name;

    private String brand;

    private String type;

    private String color;

    private String transmission;

    private int price;

    private String year;

    private MultipartFile image;

    private byte[] returnedImage;
}
